src/components/mgt-person-card/mgt-person-card.scss

```diff
@@ -31,14 +38,13 @@ $color: var(--color, #{$ms-color-neutralDark});
       line-height: 1.15;
 
       .display-name {
-        font-size: 27.3103px;
-        color: #333333;
+        font-size: $person-card-display-name-font-size;
+        color: $person-card-display-name-color;
       }
 
       .job-title {
-        font-size: 18.2069px;
-        color: #767676;
-        font-weight: 600;
+        font-size: $person-card-secondary-title-font-size;
+        color: $person-card-secondary-title-color;
       }
 
       .department {
```

Comment:
Department should also have a variable, I propose `--person-card-subtitle-*`

Reply:
Added here:
f450ea8

I feel like this might be confusing with "secondary title" and "subtitle" but I don't have better suggestions.

Reply:
Updating to: display-name, title, subtitle

dc91e65